docs/src/assets/scss/styles.scss

```diff
@@ -13,7 +12,6 @@
 @import "docs";
 @import "versions";
 @import "languages";
-
 @import "components/buttons";
 @import "components/docs-navigation";
 @import "components/toc";
```

Comment:
I agree with @snitin315. These empty lines are used to group imports, and adding comments seems unnecessary here, so perhaps we shouldn't enable this rule.